way_to_home/static/src/components/routeForm/routeForm.js

```diff
@@ -20,9 +21,38 @@ class RouteSearchForm extends Component {
     }));
   }
 
-   getCurrentPosition = () => {
-     alert('get and write current position')
-   }
+   getCurrentPosition = (props) => {
+      if (!navigator.geolocation){
+        console.log("Geolocation is not supported by your browser")
+        return;
+      }
+      self = this
+      function success(position) {
+        let latitude  = position.coords.latitude;
+        let longitude = position.coords.longitude;
+         axios.get('https://nominatim.openstreetmap.org/reverse?format=jsonv2&lat='+latitude+'&lon='+longitude+'')
```

Comment:
I think it will be better to put this URL in the variable. It will improve readability.

Reply:
ok